pkg/destroy/powervs/sshkey.go

```diff
@@ -0,0 +1,128 @@
+package powervs
+
+import (
+	"strings"
+
+	"github.com/IBM-Cloud/power-go-client/power/models"
+	"github.com/pkg/errors"
+)
+
+const sshKeyTypeName = "sshKey"
+
+// listSSHKeys lists images in the vpc.
+func (o *ClusterUninstaller) listSSHKeys() (cloudResources, error) {
```

Comment:
the keys live in your account as opposed to a VPC or Service Instance, so, s/vpc/account

Reply:
Done